### ISSUE TITLE
Feature to save the lyrics in a specifed Folder

### ISSUE THREAD

**Issue by dibyadas** [aribis369/lyrics#1]
A new feature can be added to save the lyrics in an specified folder and retrieve it later.

**Comment by aribis369** [comment]
Yes for this purpose pyfpdf ca be used . It has a good feature of converting html files to pdf files.

**Comment by dibyadas** [comment]
I think storing lyrics as pdf files will be cumbersome. It is better to keep them in a lyrics folder locally as text file and then display it in terminal when required.
